pytorch/training/docker/2.0/py3/cu118/Dockerfile.gpu

```diff
@@ -429,6 +429,17 @@ RUN /opt/conda/bin/mamba install -y python=${PYTHON_VERSION} pytorch=${PYTORCH_V
   -c nvidia \
   && /opt/conda/bin/mamba clean -afy
 
+# Override only sm-pytorch
+RUN /opt/conda/bin/mamba install --force-reinstall -y \
```

Comment:
if you installing pytorch here, then this pytorch installation should be removed to not to increase DLC image size

https://github.com/aws/deep-learning-containers/pull/3566/files#diff-bc0b77b02ee5079e588e1fd8714585bee92e4159c585a980f2f576beffbc6228L421

Reply:
I can add uninstall specifically for the torch package, we need the other packages installed